client/src/Hero.js

```diff
@@ -5,7 +5,7 @@ const Hero = () => {
     <div className="heroPageWrapper">
       <div className="heroImage" />
       <div className="zenBlossomTextContainer">
-        <div className="zenBlossomText">Zen Blossom</div>
+        <div className="zenBlossomText">Flora Denbo</div>
```

Comment:
These classnames could be updated from zenBlossomText to Flora Denbo. It may even be good to have it more generic so if any renaming does happen it never needs to be updated.